### DISCUSSION TITLE
posts/first-11-things-proxmox/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#264] [Comments]
# posts/first-11-things-proxmox/

After setting up my Proxmox servers, there are a few things I do before I use them for their intended purpose. This ranges from updates, to storage, to networking and VLANS, to uploading ISOs, to clustering, and more. Join me as we pick up where the rest of the proxmox tutorials stop, and that’s everything you need to do to make these production ready (and maybe a bonus item too).

https://technotim.live/posts/first-11-things-proxmox/

**Comment by Janov911** [comment]
Hi Tim, you might want to add the ceph.list entry in sources/list.d/ which also might need it's entry commented out.

**Comment by chieflewus** [comment]
I followed this guide and had a strange issue once I initiated the bond. My VM's stopped working. They would start and run for about 20 seconds and then shut back down. I would get an error saying timeout waiting on port 5900. 

I'm using a Dell R730, Unify equipment so it was pretty similar to your setup in the video. Once I undo the bond and take the aggregate off the switch everything works fine. I'll admit I'm extremely new to Proxmox so it might be a basic issue I'm just overlooking but I'm having some real troubles finding what could cause it.

**Comment by CPBPILOT** [comment]
So the IOMMU instructions in the video are different than what is in the written instructions. I assume the written instruction are the correct ones since you are referencing the Proxmox site

**Reply by timothystewart6** [reply]
Yes, the web site will always be the most up to date since I can't add corrections to a video!

**Comment by theonestefan** [comment]
Hi Tim. Can you please the "update-grub" command in the GRUB- section so others don't stuble over that ;-) 
Awesome work btw. Love your stuff.

**Comment by Fufs** [comment]
Hi Tim, 
I just installed PVE 8.2.2 on one of my hosts and it seems that `ceph.list` by default is also pointing to the enterprise repo. To enable updates for Ceph, you switch to one of the following:

`deb http://download.proxmox.com/debian/ceph-quincy bookworm no-subscription`
or
`deb http://download.proxmox.com/debian/ceph-reef bookworm no-subscription`

depending on the version of Ceph (Quincy is the current default)

More info available here (Sections 3.1.6 and 3.1.9): https://pve.proxmox.com/pve-docs/pve-admin-guide.html#sysadmin_package_repositories

**Comment by HeneryH** [comment]
Is there a logic error in the IOMMU writeup above?  It seems to be saying:
if x
   then a
if y
   then b
if z
   ???
-----
Checking your boot manager
Enabling PCI passthrough depends on your boot manager. You can check to see which one you are using by running

efibootmgr -v

If it returns an errors, it’s running in Legacy/BIOS with GRUB, skip to GRUB section

if it returns something like this, it’s running system-boot, skip to system-d section section

if it returns something like this.  ????
[end]

**Reply by HeneryH** [reply]
Sorry for the formatting.  Must be markdown of some sort.

**Comment by HeneryH** [comment]
In researching IOMMU and ProxMox there seems to be talk of IOMMU groups and LibVirt etc.  Is that relevant here?  None of the LibVirt stuff seems default installed on a clean ProxMox install.  I can add them all manually but I am not sure if that is heading down a bad path.
